logging/access.go

```diff
@@ -90,6 +97,14 @@ func (f *accessLogFormatter) Format(e *logrus.Entry) ([]byte, error) {
 	return []byte(fmt.Sprintf(f.format, values...)), nil
 }
 
+func stripQueryString(u string) string {
+	if i := strings.IndexRune(u, '?'); i < 0 {
```

Comment:
Is there any scenario in which we should maintain an anchor tag? I believe this will strip that too if it comes after the querystring